ResearchKit/Common/ORKStepHeaderView.m

```diff
@@ -118,6 +118,9 @@ - (instancetype)initWithFrame:(CGRect)frame {
         [_learnMoreButton setContentCompressionResistancePriority:UILayoutPriorityRequired forAxis:UILayoutConstraintAxisVertical];
         
 #ifdef LAYOUT_DEBUG
+        _captionLabel.backgroundColor = [[UIColor yellowColor] colorWithAlphaComponent:0.2];
+        _learnMoreButton.backgroundColor = [[UIColor blueColor] colorWithAlphaComponent:0.2];
+        _instructionLabel.backgroundColor = [[UIColor greenColor] colorWithAlphaComponent:0.2];
         self.backgroundColor = [[UIColor purpleColor] colorWithAlphaComponent:0.2];
```

Comment:
Second thought on this, I just found that these labels had been assigned with debug colors in method `updateConstraints`
```
        for (UIView *view in views) {
#ifdef LAYOUT_DEBUG
            view.backgroundColor = [[UIColor greenColor] colorWithAlphaComponent:0.3];
            view.layer.borderColor = [UIColor redColor].CGColor;
            view.layer.borderWidth = 1.0;
#endif
```
The newly added LAYOUT_DEBUG  color is being override, see screenshot below.

![screen shot 2015-06-12 at 10 31 11 am](https://cloud.githubusercontent.com/assets/11466704/8135726/506f50a0-10ee-11e5-984c-4407407e3a7d.png)

I like the idea to assign them with different colors, can you remove the old debug code in `updateConstraints` and move red color borders assignment code to here?